src/components/DictionaryCrud/DictionaryCrudPresentation.js

```diff
@@ -66,7 +76,7 @@ function DictionaryCrudPresentation({
     title: '',
     type: type || TYPE_WORD,
     translations: [],
-    visibility: PUBLIC,
+    visibility: isAssistant ? TEAM : PUBLIC,
```

Comment:
Could this change in default value be handled entirely in the Visibility control? We already deal with members-only sites having fewer visibility options without changing this default, so I don't think we need to have every form change the default for Assistants

Reply:
I have updated this PR to no longer update the default. Good suggestion.